bundles/org.openhab.binding.mideaac/README.md

```diff
@@ -0,0 +1,76 @@
+
+## Channels
+
+Following channels are available:
+
+| Channel               | Type                 | Description                                                                                                    | read only |
+|:----------------------|:---------------------|:---------------------------------------------------------------------------------------------------------------|:----------|
+| Power					| Switch               | Turn the AC on and off.                                                                                        |           |
+| Target temperature    | Number:Temperature   | Target temperature.                                                                                            |           |
+| Operational mode      | String               | Operational mode: OFF (turns off), AUTO, COOL, DRY, HEAT, FAN ONLY                                             |           |
+| Fan speed             | String               | Fan speed: OFF (turns off), SILENT, LOW, MEDIUM, HIGH, AUTO. Not all modes supported by all units.             |           |
+| Swing mode            | String               | Swing mode: OFF, VERTICAL, HORIZONTAL, BOTH. Not all modes supported by all units.                             |           |
+| Eco mode              | Switch               | Eco mode - Cool only (Temperature is set to 24 C (75 F) and fan on AUTO)										|			|
+| Turbo mode            | Switch               | Turbo mode, "Boost" in Midea Air app, long press "+" on IR Remote Controller. COOL and HEAT mode only.  		|           |
+| Sleep function        | Switch               | Sleep function ("Moon with a star" icon on IR Remote Controller).                                              |           |
+| ON Timer				| String               | Sets the future time to turn On the AC.																		|		    |
+| OFF Timer				| String               | Sets the future time to turn off the AC.																		|			|
+| Temperature Unit		| Switch               | Sets the evaporator display to Fahrenheit (true) or Celsius (false).											|           |
+| Screen display		| Switch			   | If device supports across LAN, turns off the LED display.														|			|
+| Indoor temperature    | Number:Temperature   | Indoor temperature measured in the room, where internal unit is installed.                                     | Yes       |
+| Outdoor temperature   | Number:Temperature   | Outdoor temperature by external unit. Some units do not report reading when off.                               | Yes       |
+| Humidity				| Number			   | If device supports, the indoor humidity.																		| Yes		|
+| Dropped Commands		| Number			   | Quality of WiFi connections - For debugging only.																| Yes		|
+
+Following items are in API but are not tested (marked as advanced and read only channels):
+
+- Appliance error
+- Auxiliary heat
+- Alternate Target Temperature
+
```

Comment:
Please also provide Full example: thing and item are mandatory, the sitemap example is optional. You could check another binding as inspiration.

Reply:
I guess I missed the OH guidance regarding the ReadMe file ;-). Also had not tested textual configuration. Needed to read up on markdown formatting and test the textual Thing, Item and Sitemap examples. I needed switch the AC version parameter to a configuration item and update the thing-type and mideaac.properties